dcc-portal-ui/app/scripts/analysis/js/analysis.js

```diff
@@ -166,7 +166,7 @@
           $scope.analysisResult = null;
           $timeout(function() {
             $scope.analysisResult = data;
-          }, 150);
+          }, 250);
```

Comment:
There appears to be a race condition with the `d3` `.transition()` in that if I modify the data or trigger any re-rendering too quickly while it is still firing that styles do not get applied properly. 

Commenting and testing a different solution would be preferable yes.